testsuite/lint.py

```diff
@@ -15,7 +15,7 @@ def main() -> None:
 @main.command()
 @click.argument("paths", nargs=-1)
 def helm(paths: Tuple[str]) -> None:
-    shell = LocalShell(True)
+    shell = LocalShell()
```

Comment:
Please add a call to init_logging with debug level to keep the current level of logging

Reply:
I can do that in another commit to unbreak the tests. I'll need to do some refactoring first.